src/main/kotlin/fi/metatavu/muisti/devices/ExhibitionDeviceGroupController.kt

```diff
@@ -102,6 +121,45 @@ class ExhibitionDeviceGroupController {
     return result
   }
 
+  /**
+   * Copies device group including all it's contents (devices, content versions, group content versions and pages).
+   *
+   * @param sourceDeviceGroup source device group
+   * @param namePrefix prefix for copied resource names
+   * @param creatorId id of user that created the copy
+   * @return copied device group
+   */
+  fun copyDeviceGroup(
+    sourceDeviceGroup: ExhibitionDeviceGroup,
+    namePrefix: String,
+    creatorId: UUID
+  ): ExhibitionDeviceGroup {
+    logger.info("Creating copy of device group {}", sourceDeviceGroup.id)
```

Comment:
Maybe use debug log level?

This comment applies to all logger.info lines in this file, too lazy to comment same thing to multiple lines :)